HeadstormChallenge/ConvertDB.py

```diff
@@ -0,0 +1,39 @@
+#Jared Gatlin
+
+import json
+import sqlite3
+
+#Connects to database already created with sqlite3
+#Already created Person table in the DB
+connection = sqlite3.connect('data.db')
```

Comment:
What happens if and when the connection fails?

Reply:
Hi Faisal,
In this exercise I did not have a check depending on if this connection fails. However, in order to compensate for that error, I should have had a check to determine if the connection failed and if so, create a new database and connect to that one.